java/src/org/openqa/selenium/grid/router/httpd/RouterFlags.java

```diff
@@ -66,6 +66,13 @@ public class RouterFlags implements HasRoles {
   @ConfigValue(section = NETWORK, name = "sub-path", example = "my_company/selenium_grid")
   public String subPath;
 
+  @Parameter(
+    names = {"--disable-ui"},
+    arity = 1,
+    description = "Disable the UI on the Hub/Router/Standalone")
+  @ConfigValue(section = NETWORK, name = "disable-ui", example = "true")
```

Comment:
Shouldn't the section be "router"? I might be missing something here.

Reply:
Thanks @pujagani for the feedback, I made the changes based on this.